### ISSUE TITLE
SVT : Eclipse: Jav Microprofile : Open application view fails for running project

### ISSUE THREAD

**Issue by mmulholl** [eclipse/codewind#510]
**Codewind version:**E   Codewind tech preview	0.4.0.v201909191538	
**OS:** Mac

**Che version:**
**IDE extension version:**
**IDE version:** Eclipse Comitter 2019-06
**Kubernetes cluster:**

**Description:**
When opening an application, application monitor, or performance monitor views can result in this error: It appears to be intermittent but regular:

![image](https://user-images.githubusercontent.com/16086270/65428252-0f446a00-dde2-11e9-85f8-056d0da1f594.png)

**Steps to reproduce:**
Craete a java micropofile project, wait for it to be running and open the views.

**Workaround:**

**Comment by mmulholl** [comment]
@jagraj @eharris369

**Comment by mmulholl** [comment]
Yes, it is a problem with the app state jumping around but as you see in the picture above the project status is "running" at the time of the error. It may be a 2 stage build and you need to go through stopped -> running -> stopped -> running before opening the view but that is bad and a regression, running should only show if the project is ready.

**Comment by mmulholl** [comment]
Any plans to address this? Add some intelligence to codewind to hold off until things are stable?

**Comment by rajivnathan** [comment]
@mmulholl Are you still seeing this issue?
